client/my-sites/domains/domain-management/components/primary-domain-selector/style.scss

```diff
@@ -0,0 +1,17 @@
+.domains-set-primary-address {
+	&.form-fieldset {
+		margin-bottom: 2em;
+		box-shadow: 0 0 0 1px var(--color-border-subtle);
+		padding: 24px;
+	}
```

Comment:
If I wanted to be incredibly nit-picky (in other words, feel free to ignore!), you could wrap the `FormFieldSet` in a `Card` and then you wouldn't need these styles.

Reply:
I think this probably should be made a Card now since the button effectively just makes it a Card.